src/opnsense/mvc/app/views/OPNsense/Core/halt.volt

```diff
@@ -0,0 +1,22 @@
+<p><strong>{{ lang._('Are you sure you want to power off the system?') }}</strong></p>
```

Comment:
copyright header missing (since rewrite it's a new file so no need to drag other stuff around)